### PR TITLE
Updating the `az datafactory` documentation - not all functionality is available right now

### PR DESCRIPTION
**Pull request by mindlessroman**
Hey folks -

Linking to https://github.com/Azure/azure-cli/issues/23844 - Updating the documentation to acknowledge that there is not 100% feature support on the AzCLI for all Data Factory components.

### CODE REVIEW

**Review by chasewilson — CHANGES_REQUESTED** [review]
Thanks for the PR! I just have a couple of comments and then we should be able to get this merged :)

File: latest/docs-ref-autogen/datafactory.yml [hunk 1]
```diff
@@ -4,8 +4,13 @@ name: az datafactory
...
 
-  > This reference is part of the **datafactory** extension for the Azure CLI  (version 2.15.0 or higher).  The extension will automatically install the first time you run an **az datafactory** command.  [Learn more](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview) about extensions.
-summary: Manage Data Factory.
+  > This reference is part of the **datafactory** extension for the Azure CLI  (version 2.15.0 or higher).  The extension will automatically install the first time you run an **az datafactory** command. Learn more about [Azure CLI extensions](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview). Learn more about the [`az datafactory` extension.](https://github.com/Azure/azure-cli-extensions/tree/main/src/datafactory)
+summary: >-
+  Manage Data Factory.
+
+  > [!Note]
+
+  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, one can use the PowerShell module for [Data Factory.](https://docs.microsoft.com/en-us/powershell/module/az.datafactory/?view=azps-8.3.0)
```
[the remark this answers is not in the document]
Comment (mindlessroman) [reply latest/docs-ref-autogen/datafactory.yml:13]: Good point on pointing out what's missing. It seems to be Data Flows primarily but it's worth confirming

As for a timeline, I'm not sure. I'm investigating how to add it, but I don't know how that will be prioritized in the next while.

[the remark this answers is not in the document]
Comment (chasewilson) [reply latest/docs-ref-autogen/datafactory.yml:13]: No worries about the timeline. If we don't have any info then it's better not to muddy the waters :) Thanks for the quick response and if we can get a little more guidance on what's missing, we'll get this merged in.

[the remark this answers is not in the document]
Comment (mindlessroman) [reply latest/docs-ref-autogen/datafactory.yml:13]: Sounds good! I'll try to firm up what's missing and update here!

[the remark this answers is not in the document]
Comment (mindlessroman) [reply latest/docs-ref-autogen/datafactory.yml:13]: Basing the findings by comparing what's available on https://docs.microsoft.com/en-us/powershell/module/az.datafactory/?view=azps-8.3.0#data-factories

File: latest/docs-ref-autogen/datafactory.yml [hunk 2]
```diff
@@ -4,8 +4,13 @@ name: az datafactory
...
   > [!Note]
 
-  > This reference is part of the **datafactory** extension for the Azure CLI  (version 2.15.0 or higher).  The extension will automatically install the first time you run an **az datafactory** command.  [Learn more](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview) about extensions.
-summary: Manage Data Factory.
+  > This reference is part of the **datafactory** extension for the Azure CLI  (version 2.15.0 or higher).  The extension will automatically install the first time you run an **az datafactory** command. Learn more about [Azure CLI extensions](https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview). Learn more about the [`az datafactory` extension.](https://github.com/Azure/azure-cli-extensions/tree/main/src/datafactory)
+summary: >-
+  Manage Data Factory.
+
+  > [!Note]
+
```
[the remark this answers is not in the document]
Comment (mindlessroman) [reply latest/docs-ref-autogen/datafactory.yml:12]: So it seems that this new line is necessary for the note formatting.

File: latest/docs-ref-autogen/datafactory.yml [hunk 3]
```diff
@@ -4,414 +4,421 @@ name: az datafactory
...
-    description: ''
-  - name: --factory-vsts-configuration --vsts-config
-    summary: Factory's VSTS repo information.
-    description: >-
-      Usage: --factory-vsts-configuration project-name=XX tenant-id=XX type=XX account-name=XX repository-name=XX collaboration-branch=XX root-folder=XX last-commit-id=XX
+  > [!Note]
 
+  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, one can use the PowerShell module for [Data Factory.](https://docs.microsoft.com/en-us/powershell/module/az.datafactory/?view=azps-8.3.0)
+  > Currently not supported: Data Flow commands; Activity Window; Data Factory
+  Gateways, Hubs; Saving Logs
```
Comment (chasewilson) [inline latest/docs-ref-autogen/datafactory.yml:15]:
```suggestion
  > Currently not supported: Data Flow commands; Activity Window; Data Factory Gateways, Hubs; Saving Logs.
  
```

**Review by chasewilson — CHANGES_REQUESTED** [review]

**Review by chasewilson — APPROVED** [review]

File: latest/docs-ref-autogen/datafactory.yml [hunk 4]
```diff
@@ -4,414 +4,425 @@ name: az datafactory
...
-    description: >-
-      Usage: --factory-vsts-configuration project-name=XX tenant-id=XX type=XX account-name=XX repository-name=XX collaboration-branch=XX root-folder=XX last-commit-id=XX
+  > [!Note]
 
+  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, please use the PowerShell module for [Data Factory](https://docs.microsoft.com/powershell/module/az.datafactory). Currently not supported: 
+  - Data Flow commands 
+  - Activity Window
+  - Data Factory Gateways
+  - Hubs
+  - Saving Logs
```
Comment (chasewilson) [inline latest/docs-ref-autogen/datafactory.yml:18]:
```suggestion
  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, please use the PowerShell module for [Data Factory](https://docs.microsoft.com/powershell/module/az.datafactory). Currently not supported: Data Flow commands, Activity Window, Data Factory Gateways, Hubs, Saving Logs
```

**Comment by mindlessroman** [comment]
@chasewilson - I don't have the ability to merge this, do you?

**Comment by chasewilson** [comment]
Hey Hannah, I do :) I was just updating a few things for formatting and visual assistance but should have it merged by eod :)

File: latest/docs-ref-autogen/datafactory.yml [hunk 5]
```diff
@@ -4,414 +4,420 @@ name: az datafactory
...
-  - name: --factory-resource-id
-    summary: The factory resource id.
-    description: ''
-  - name: --factory-vsts-configuration --vsts-config
-    summary: Factory's VSTS repo information.
-    description: >-
-      Usage: --factory-vsts-configuration project-name=XX tenant-id=XX type=XX account-name=XX repository-name=XX collaboration-branch=XX root-folder=XX last-commit-id=XX
+  > [!Note]
 
+  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, please use the PowerShell module for [Data Factory](https://docs.microsoft.com/powershell/module/az.datafactory). Currently not supported: Data Flow commands, Activity Window, Data Factory Gateways, Hubs, Saving Logs
```
Comment (chasewilson) [inline latest/docs-ref-autogen/datafactory.yml:13]:
```suggestion
  > Not all Data Factory functionality is supported in the CLI at this time. For any missing functionality, please use the PowerShell module for [Data Factory](https://docs.microsoft.com/powershell/module/az.datafactory). Currently not supported: **Data Flow commands**, **Activity Window**, **Data Factory Gateways**, **Hubs**, and **Saving Logs**
```

**Comment by chasewilson** [comment]
Closed to kick of build

**Comment by chasewilson** [comment]
A huge thank you @mindlessroman for this great update :)

**Comment by mindlessroman** [comment]
Thank you, thank you @chasewilson !

**Pull request merged**